### PR TITLE
Household Service Documentation

### PR DESCRIPTION
**Pull request by justinstroehleFH**

### CODE REVIEW

File: documentation/implementation/household-service/Household_Service.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+The `Postgres` adapters implement the repository interface to save and fetch from our Postgres database. Internally, they delegate most of the requests to the respective JPA repositories, which are provided by Spring to simplify basic operations to the underlying database.
+
+#### Redis Adapter
+
+Apart from that we also implemented a `Redis-Adapter` that publishes for `HouseholdEvents`.
+
+#### Rest Adapter
+
+The Rest adapters allow to access the service via HTTP. The applications offers endpoints to create households and users, list the households and users, map users to households and assign meters to housholds.
```
Comment (xBlade58) [inline documentation/implementation/household-service/Household_Service.md:48]:
```suggestion
The Rest adapters allow to access the service via HTTP. The applications offers endpoints to create households and users, list the households and users, map users to households and assign meters to households.
```

**Review by meroez — COMMENTED** [review]
Can you please also provide an example for a `HouseholdEvent` sent by Redis?

**Review by bellerstefan — COMMENTED** [review]
Ok from my point of view - to fix the changes from mert

File: documentation/implementation/images/Household_Service_Architecture.png
Comment (bellerstefan) [inline documentation/implementation/images/Household_Service_Architecture.png:1]: geplant dass redis pfeil ins nichts geht?

**Review by bellerstefan — COMMENTED** [review]
see hexagon png issue

File: documentation/implementation/images/Household_Service_Architecture.png
Comment (justinstroehleFH) [reply documentation/implementation/images/Household_Service_Architecture.png:1]: HomerBrainGIF

**Review by bellerstefan — DISMISSED** [review]
ok for me

**Review by meroez — APPROVED** [review]

**Pull request merged**